app/.gitignore

```diff
@@ -24,3 +24,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/.vscode
```

Comment:
Maybe this is a typo, should it be `.vscode/` to prevent .vscode/settings.json from getting added?

Reply:
We have two .vscode folders in this repo - at the root level and app level. One of them (I forgot which) was being included in my commits. This seemed to solve it. I will delete it and put it in the root level .gitignore ^_^